cv_next/app/cv/[cvId]/components/commentSection/comment.tsx

```diff
@@ -303,23 +305,27 @@ export default function Comment({
     };
 
     try {
-      mutate(comment.document_id, optimisticData, {
-        optimisticData: optimisticData(commentsOfComment),
-        rollbackOnError: (error) => {
-          // Type-checking for error
-          if (error instanceof Error) {
-            return error.name !== "AbortError";
-          }
-          // Rollback if error is not an instance of Error
-          return true;
+      mutate(
+        comment.document_id,
+        async () => {
+          await addComment(commentToAdd);
+          return optimisticData(commentsOfComment);
         },
-      });
-      await addComment(commentToAdd);
-      mutate(comment.document_id);
+        {
+          optimisticData: optimisticData(commentsOfComment),
+          rollbackOnError: (error) => {
+            if (error instanceof Error) {
+              return error.name !== "AbortError";
+            }
+            return true;
+          },
+          revalidate: true,
+        }
+      );
     } catch (error) {
-      mutate(comment.document_id);
+      router.push("/inactive");
```

Comment:
I think we shouldn't redirect to "inactive" here, we want to do some app-level error handling if we suspect inactive user (practically: create custom error that you'll throw e.g. here and catch it at the top level).

Reply:
If I remember correctly we decided to redirect to the login page with a next param but we just don't have it implemented here(we have it in the SSR branch that also fixed some problems with the flow of authorization itself). I'll add a TODO on relevant places.
Should I still do a custom error that'll catch this and add the TODO there?

Reply:
Nah it's fine leave it as TODO and fix it in a later branch